sdk/eventhub/Azure.Messaging.EventHubs/tests/Infrastructure/EventProcessorManager.cs

```diff
@@ -148,8 +148,8 @@ public Task StopAllAsync()
         }
 
         /// <summary>
-        ///   Waits until the partition load distribution is stabilized.  Throws a <see cref="TimeoutException"/> if
-        ///   the load takes too long to stabilize.
+        ///   Waits until the partition load distribution is stabilized.  Throws an <see cref="OperationCanceledException"/>
+        ///   if the load takes too long to stabilize.
         /// </summary>
         ///
         /// <returns>A task to be resolved on when the operation has completed.</returns>
```

Comment:
Added it to the delay. Partition Manager's methods do not support cancellation tokens. It should be simple to include it in this PR if you think we should.

Reply:
Partition Manager is user-supplied or one of our constructs?   If I recall, the normal use case is that it hangs out in partition context and is invoked by the user right?   I would think as an async method, it should have cancellation in there... but not a priority for preview 3.  Definitely something we want to get in for the actual blob storage thinger.

Reply:
It can be user-supplied, but we also provide two existing implementations (InMemory and BlobStorage).

It has three methods (all of them async):
 - ListOwnership: used by us for load balancing. I believe the user could make use of it as well.
 - ClaimOwnership: used by us for load balancing. I don't think it's useful for the user, as it requires a list of `PartitionOwnership`, a class with a `protected internal` constructor.
 - UpdateCheckpoint: indirectly called by the user via `PartitionContext`. We don't call it by ourselves. The user shouldn't call it directly, as it requires a `Checkpoint` instance, something they can't create.

I'm not sure if this gives you enough context to think about that, but we can discuss it for the next preview.

Reply:
Yeah, I'd definitely say that, based on the context, we should make sure that we bubble cancellation down into it and respect it at that level.   As you said,  conversation for P4.    If you would be so kind, would you open an issue to track?